src/js/views/patients/worklist/worklist_views.js

```diff
@@ -322,10 +298,6 @@ const ListView = CollectionView.extend({
     if (!this.isAttached()) return;
     this.triggerMethod('update:listDom', this);
   },
-  viewFilter({ model }) {
```

Comment:
@LisaManresa I'm walking this back.  We don't current filter the worklists after changes.. only doing that on patient dashboards.  Any reason to introduce it here to only flows?

Reply:
This was in place to ensure that when an item's state is changed from `done` it is visually removed from the list (on the "done < 30 days" list only). But perhaps the better approach would be to remove them from the collection in the list view on `change:visible`.

Reply:
we don't currently visually remove anything from any worklist if it's status changes

Reply:
Ah ok. Then yeah, this can go.